### PR TITLE
Adds Harmonicas To Cell Lockers

### PR DESCRIPTION
**Pull request by AffectedArc07**
Prisoners can now play the blues on harmonicas to pass the time

:CL: AffectedArc07
add: Cell lockers now have harmonicas so prisoners can pass the time with the blues
/ :CL:

### CODE REVIEW

**Comment by Ralta** [comment]
pls no

**Comment by Birdtalon** [comment]
A funny sentiment but they'll be locked in the lockers by the officers.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -380,6 +380,7 @@
 		new /obj/item/clothing/shoes/orange(src)
 		new /obj/item/weapon/card/id/prisoner/random(src)
 		new /obj/item/device/radio/headset(src)
+		new /obj/item/device/instrument/harmonica
```
Comment (Birdtalon) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: `new /obj/item/device/instrument/harmonica(src)`

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 2]
```diff
@@ -380,6 +380,7 @@
 		new /obj/item/clothing/shoes/orange(src)
 		new /obj/item/weapon/card/id/prisoner/random(src)
 		new /obj/item/device/radio/headset(src)
+		new /obj/item/device/instrument/harmonica(src)
```
Comment (Birdtalon) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: You only need one newline really.

Comment (AffectedArc07) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: I wasnt aware

**Comment by Spartan6** [comment]
Put a single one in Perma and a single one in solitary.

**Comment by Spacemanspark** [comment]
I don't think we need to clutter up the lockers.

**Comment by Fox-McCloud** [comment]
I'm kinda with @Spacemanspark 

To be honest, I can see just as many using this to be annoying rather than to just pass the time for boredom.

Also, it's a window breaker.

**Comment by AffectedArc07** [comment]
So are shoes.

**Comment by KasparoVy** [comment]
Make a plastic harmonica subtype that has no force or throwforce. ;)

**Comment by AffectedArc07** [comment]
Closed due to unforeseen circumstances

**Comment by taukausanake** [comment]
Noooooooo, we need more music!

**Pull request closed without merging**